apps/web/src/components/NotificationDialog/NotificationDialog.tsx

```diff
@@ -146,8 +146,8 @@ const NotificationDialog = () => {
                     {intl.formatMessage(
                       {
                         defaultMessage:
-                          "Welcome to your notification panel. Click or tap a notification to be taken to the relevant page. Use the <icon></icon> icon to mark a specific notification as read, pin it, or delete it.",
```

Comment:
Having removed the icon, don't think you need the circled bit anymore

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/4d7fc086-3ded-47b2-8efa-d279dba74e2d)

Reply:
Nice catch! Fixed by [0d08784](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10853/commits/0d087840cf042fb15c721b2fdcc17cb16cab4384)